examples/tokenvm/actions/mint_asset.go

```diff
@@ -74,10 +74,15 @@ func (m *MintAsset) Execute(
 		}, nil
 	}
 	newSupply, err := smath.Add64(supply, m.Value)
+
+	if maxSupply > 0 && newSupply > maxSupply {
```

Comment:
remove `maxSupply > 0` here. If unlimited supply is allowed, it will be `consts.MaxUint64`.